TGC/common/cb_types.txt

```diff
@@ -8096,3 +8097,39 @@ USS_monroe = {
 	po_transfer_provinces = yes
 	po_release_puppet = yes
 }
+
+enforce_clayton_treaty = {
+	sprite_index = 19
+	is_triggered_only = yes
+	months = 12
+	crisis = no
+	constructing_cb = no
+
+	badboy_factor = 0
+	prestige_factor = 1
+	peace_cost_factor = 15
+	penalty_factor = 1
+
+	break_truce_prestige_factor = 1
+	break_truce_infamy_factor = 1
+	break_truce_militancy_factor = 1
+	truce_months = 48
+
+	good_relation_prestige_factor = 1
+	good_relation_infamy_factor = 1
+	good_relation_militancy_factor = 1
+
+	always = yes
+
+	can_use = { year=9999 } # I don't know how to remove this properly
```

Comment:
what is the purpose of this

Reply:
I wish to disable this cb from being used except by event. I assume there is a better way to do this?

Reply:
always = no
two lines above

Reply:
Do we even need to have a CB for this or can we use something like cut_down_to_size

Reply:
This is supposed to be a relatively minor war so I didn't want to have any of the cut down to size side effects. Something like war reparations would be fine, but I like the ability to have a custom war name so it's more clear to players who aren't involved in the war itself.

Reply:
Custom war names are possible with generic wargoals, but since this is an always = no war it won't affect performance so it's okay.